gitkraken-client/workspaces.md

```diff
@@ -152,17 +124,82 @@ The Team Pull Requests section has the following columns:
+- Fetch: fetch all selected repositories.
+- Open in GitKraken Client or an external editor: open all selected repositories in GitKraken or in your [default editor](/start-here/preferences/#external-editor).
+- Remove: remove all selected repositories from the workspace.
+
+### Create a Cloud Workspace from a Local Workspace
+
+You can create a Cloud Workspace from a Local Workspace which will enable more visibility into your pull requests, issues, and share your Workspace with teams.
+
+To do this, select your local workspace to open it and then select `Create cloud workspace`. From here, you will need to select the provider for your new Cloud Workspace, select the repositories you would like added, and you can even add more repositories to this workspace from your provider.
+
+<img src="/wp-content/uploads/create-local-workspace.png" class="img-bordered img-responsive center">
+
 <div class='callout callout--warning'>
     <p>
-        <strong>Note:</strong> This Team Overview section is in Preview mode, and we’d love to hear your thoughts and feedback. Just click on the “Provide feedback on this view” prompt in the upper left of the Overview page to tell us what you think.
+        <strong>Note:</strong>
+            To create a Cloud Workspace, the integration for the desired service must be connected under <kbd>Preferences > Integrations</kbd>.
     </p>
 </div>
 
+***
+
+## Edit a workspace
+
+Edit a workspace by selecting the ellipsis <i class="fas fa-ellipsis-v"></i> icon by the Workspace name.
+
+<img src="/wp-content/uploads/edit-a-workspace.png" srcset="/wp-content/uploads/edit-a-workspace@2x.png" class="img-bordered img-responsive center">
+
+You can adjust the content you want displayed for each view by selecting the settings <i class="fas fa-cog"></i>.
+
+<img src="/wp-content/uploads/configure-focus-view.png" class="img-bordered img-responsive center">
+
+***
+
+## View Repository Details
+
+You may select the <i class="fa-solid fa-list"></i> icon to open the repository details.
+
+<img src="/wp-content/uploads/repository-details.png" srcset="/wp-content/uploads/repository-details@2x.png" class="img-bordered img-responsive center">
+
+***
+
+## Repository statuses
+
+Workspaces expose the state of all repositories so you can see the last checked-out branch, how many commits are ahead or behind the remote, and your work in progress with counts of modified added, deleted, and renamed files.
+
+<img src="/wp-content/uploads/repos-status.png" srcset="/wp-content/uploads/repos-status@2x.png" class="img-bordered img-responsive center">
+
+***
+
+## Workspace breadcrumb in toolbar
+
+The option to remove the Workspace breadcrumb in the toolbar can be toggled under <kbd> Preferences > UI Customization > _Show Workspace breadcrumb in toolbar_</kbd>.
+
+<img src="/wp-content/uploads/breadcrumb-setting.png" srcset="/wp-content/uploads/breadcrumb-setting@2x.png" class="img-bordered img-responsive center">
```

Comment:
No comment about this code, but I suspect I'll have more feedback when we push this production tomorrow morning.